spec/index.adoc

```diff
@@ -1654,6 +1654,28 @@ There is no restriction on who can invoke this method.
 
 This method takes no input and returns 32 pseudo-random bytes to the caller. The return value is unknown to any part of the IC at time of the submission of this call. A new return value is generated for each call to this method.
 
+[#ic-get_ecdsa_public_key]
+=== IC method `get_ecdsa_public_key`
+
+This method returns the threshold ECDSA public key for the given canister using the given derivation path. If the `canister_id` is unspecified, it will default to the canister id of the caller.
```

Comment:
It should be transparent to the canister whether this is a threshold key.

```suggestion
This method returns the ECDSA public key for the given canister using the given derivation path. If the `canister_id` is unspecified, it will default to the canister id of the caller.
```